examples/sn-blog-gatsby/src/layouts/index.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from 'react'
 import { Helmet } from 'react-helmet'
 import LayoutMain from '../components/layout-main'
 import LayoutRoot from '../components/layout-root'
```

Comment:
these components are does nothing important what would change the behavior or the style of the application
I think they are safe to remove